src/lib/userStorage/UserStorageClass.js

```diff
@@ -786,7 +786,7 @@ export class UserStorage {
       FeedItemType.EVENT_TYPE_SENDBRIDGE,
     ]
 
-    if (counterPartyEvents.includes(type) && (!counterPartyFullName || !counterPartySmallAvatar)) {
```

Comment:
that doesnt seem correct. the small avatar is used in feed as the image.
I understand that TXs that are not p2p won't have a small avatar, so we probably need that type of filter

Reply:
let make sure we have some kind of retry for the regular getCounterParty